project-koku/koku
# COST-261 bubble up costs for sub_orgs when group_by is used.

Testing:

1. Fill database using make command: `make load-aws-org-unit-tree`

2. Go to this endpoint: [http://127.0.0.1:8000/api/cost-management/v1/reports/aws/costs/?group_by[org_unit_id]=R_001](http://127.0.0.1:8000/api/cost-management/v1/reports/aws/costs/?group_by[org_unit_id]=R_001) and note the total for infrastructure and cost.

3. Go to the following endpoints and add the values for total infrastructure and cost up and they should match the total from above: 

- [http://127.0.0.1:8000/api/cost-management/v1/reports/aws/costs/?group_by[org_unit_id]=OU_001](http://127.0.0.1:8000/api/cost-management/v1/reports/aws/costs/?group_by[org_unit_id]=OU_001)

- [http://127.0.0.1:8000/api/cost-management/v1/reports/aws/costs/?group_by[org_unit_id]=OU_002](http://127.0.0.1:8000/api/cost-management/v1/reports/aws/costs/?group_by[org_unit_id]=OU_002)

- [http://127.0.0.1:8000/api/cost-management/v1/reports/aws/costs/?filter[org_unit_id]=R_001](http://127.0.0.1:8000/api/cost-management/v1/reports/aws/costs/?filter[org_unit_id]=R_001)
